app/models/user.rb

```diff
@@ -106,12 +118,14 @@ def balance
     bal - total_pending
   end
 
+  def avatar(*)
+    "https://zpi.zeus.gent/image/#{zauth_id || 0}"
```

Comment:
Should this be a config option? @TomNaessens where in the config should you put functions?